lib/checkstl.cpp

```diff
@@ -1373,7 +1373,7 @@ void CheckStl::if_find()
             continue;
 
         const Token *conditionStart = scope.classDef->next();
```

Comment:
I don't understand how Coverity comes to the conclusion that `conditionStart` must be non-null?

Reply:
Because we are calling conditionStart->astOperand2() without checking if conditionStart is NULL.
```
        const Token *conditionStart = scope.classDef->next();
        if (Token::simpleMatch(conditionStart->astOperand2(), ";"))
            conditionStart = conditionStart->astOperand2();

```